infrastructure/src/main/java/org/corfudb/infrastructure/LogReplicationServer.java

```diff
@@ -111,7 +112,7 @@ private void handleLrEntryRequest(@Nonnull RequestMsg request,
                                       @Nonnull IServerRouter router) {
         log.trace("Log Replication Entry received by Server.");
 
-        if (isLeader(request, ctx, router, true)) {
+        if (isStandby.get() && isLeader(request, ctx, router, true)) {
```

Comment:
I know this is not from your change, just noticed while reading the method's comment. It should be "get log-data entries and apply them" instead of "send by the log-data entries" right?

Reply:
you are right, I changed it to say " send an ACK back on processing the msg".